OTAcceleratorCore/OTMultiPartyCommunicator.m

```diff
@@ -385,10 +394,27 @@ - (void)session:(OTSession *)session streamDestroyed:(OTStream *)stream {
     }
 }
 
+- (void)  session:(OTSession*) session
```

Comment:
there is a double space between (void)   session, I would look the best if there's no space :)